src/midi/midi_message_handler.rs

```diff
@@ -0,0 +1,5 @@
+use crate::{Message, Result};
+
+pub trait MidiMessageHandler: Send + Sync {
```

Comment:
I ended up not using this, this was when I was trying to "register listeners" with `MidiMessagePublisher`